governance/board-sync.py

```
@@ -0,0 +1,354 @@
+
+def scan_cards(repos):
+    """全部 active 仓 open issue 且带 state:* 标签 → 卡列表（label 是唯一判据）。"""
+    cards = []
+    for repo in repos:
+        page = 1
+        while True:
+            batch = api_get(f"/repos/{ORG}/{repo}/issues?state=open&per_page=100&page={page}")
+            for it in batch:
+                if "pull_request" in it:  # issues 端点混入 PR——不是卡
+                    continue
+                sl = [l["name"] for l in it.get("labels", []) if str(l.get("name", "")).startswith("state:")]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)